validator/validation/validation.py

```diff
@@ -168,47 +174,77 @@ def create_pytesmo_validation(validation_run):
         if validation_run.anomalies == ValidationRun.CLIMATOLOGY:
             # make sure our baseline period is in UTC and without timezone information
-            anomalies_baseline=[validation_run.anomalies_from.astimezone(tz=pytz.UTC).replace(tzinfo=None),
-                                validation_run.anomalies_to.astimezone(tz=pytz.UTC).replace(tzinfo=None)]
-            reader = AnomalyClimAdapter(reader, columns=[dataset_config.variable.pretty_name], timespan=anomalies_baseline)
+            anomalies_baseline = [validation_run.anomalies_from.astimezone(tz=pytz.UTC).replace(tzinfo=None),
+                                  validation_run.anomalies_to.astimezone(tz=pytz.UTC).replace(tzinfo=None)]
+            reader = AnomalyClimAdapter(reader, columns=[dataset_config.variable.pretty_name],
+                                        timespan=anomalies_baseline)
 
-        if ((validation_run.reference_configuration) and
-            (dataset_config.id == validation_run.reference_configuration.id)):
+        if (validation_run.reference_configuration and
+                (dataset_config.id == validation_run.reference_configuration.id)):
             # reference is always named "0-..."
             dataset_name = '{}-{}'.format(0, dataset_config.dataset.short_name)
         else:
             dataset_name = '{}-{}'.format(ds_num, dataset_config.dataset.short_name)
             ds_num += 1
 
-        ds_list.append( (dataset_name, {'class': reader, 'columns': [dataset_config.variable.pretty_name]}) )
+        ds_list.append((dataset_name, {'class': reader, 'columns': [dataset_config.variable.pretty_name]}))
 
-        if ((validation_run.reference_configuration) and
-            (dataset_config.id == validation_run.reference_configuration.id)):
+        if (validation_run.reference_configuration and
+                (dataset_config.id == validation_run.reference_configuration.id)):
             ref_name = dataset_name
             ref_short_name = validation_run.reference_configuration.dataset.short_name
 
-        if ((validation_run.scaling_ref) and
-            (dataset_config.id == validation_run.scaling_ref.id)):
+        if (validation_run.scaling_ref and
+                (dataset_config.id == validation_run.scaling_ref.id)):
             scaling_ref_name = dataset_name
 
     datasets = dict(ds_list)
     ds_num = len(ds_list)
 
     period = None
     if validation_run.interval_from is not None and validation_run.interval_to is not None:
-        ## while pytesmo can't deal with timezones, normalise the validation period to utc; can be removed once pytesmo can do timezones
+        # while pytesmo can't deal with timezones, normalise the validation period to utc; can be removed once pytesmo can do timezones
         startdate = validation_run.interval_from.astimezone(UTC).replace(tzinfo=None)
         enddate = validation_run.interval_to.astimezone(UTC).replace(tzinfo=None)
         period = [startdate, enddate]
```

Comment:
Pytesmo should now be able to handle timezones, I think. We should remove this, but maybe in another PR

Reply:
I've tried, but doesn't work:
```
        startdate = validation_run.interval_from
        enddate = validation_run.interval_to
        period = [startdate, enddate]
```

Reply:
Okay, I will try to look into this.